src/ocean/task/Task.d

```diff
@@ -299,8 +310,8 @@ public abstract class Task : ISuspendable
     public void suspend ( )
     {
         assert (this.fiber !is null);
```

Comment:
shouldn't it be replaced to `verify()` too?

Reply:
No, `task` package is currently the only one which uses both as intended (depending on what kind of condition is protected against), contrary to rest of ocean which was converted 1-to-1.

I will likely have a full re-read of all scheduler source code soon and will also re-evaluate all assert/verify usage cases.

Reply:
ok, good to know that